packages/client/libclient-py/quickmpc/share.py

```diff
@@ -98,7 +98,7 @@ def __sharize_1dimension_int(secrets: List[int], party_size: int = 3) \
         """ 1次元リストのシェア化 """
         rnd: RandomInterface = ChaCha20()
         secrets_size: int = len(secrets)
-        max_val = max(secrets) * 2
+        max_val = (max(secrets)+1) * 2
```

Comment:
@mdonaka 
What is the reason for +1, 0 avoidance?

Reply:
Yes. It is 0 avoidance.
The range is subject to reconsideration and will be considered separately.